clang/lib/DPCT/Error.h

```diff
@@ -64,6 +64,7 @@ enum ProcessStatus {
   MigarteBuildScriptIncorrectUse = -50,
   MigarteBuildScriptAndMigarteBuildScriptOnlyBothUse = -51,
   MigrationErrorNoExplicitInRootAndCMakeScript = -52,
+  MigrationErrorOptionConflict = -53,
```

Comment:
There is already an option `MigrationErrorConflictOptions`. Can we use it directly? now it only use in query-api-mapping.